lang/cs/Org.Apache.REEF.Common/metrics/Api/IMetricsSystem.cs

```diff
@@ -66,7 +66,8 @@ public interface IMetricsSystem : IObservable<IMetricsRecord>
         /// before returning. However, if it cannot be done in reasonable time 
         /// it is ok to return to the caller before everything is done. 
         /// </summary>
-        void PublishMetricsNow();
+        /// <param name="all">Whether to also publish metrics that did not change.</param>
+        void PublishMetricsNow(bool all);
```

Comment:
Please use a more descriptive parameter name, like `publishUnchangedMetrics` (otherwise it's not intuitive what's the behavior of not-all scenario)